collections/_development_standards/new-solutions.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title:  "New Solutions"
+category: development
+---
+
+## New solutions
+
+Below are some considerations that we should be taking into account when creating new solutions.
+
+### Which .Net version should I use
+
+In the main new solutions should be using the latest long-term support version available at the time. It also needs to be supported by our build servers so a check-in with the DevOps team before starting is always useful.
```

Comment:
Could we use stronger wording here?  Maybe something like `New solutions should be using the latest long-term support version available at the time, exceptions should only be made where a feature available in a newer version is essential to deliver required functionality.`